test/sample_test.yaml

```diff
@@ -76,9 +76,6 @@ spec:
               - retry
               - preemptible_tpu_gpu
               - volume_snapshot_ops
-              - loop_output
-              - loop_parameter
-              - loop_static
```

Comment:
the three samples are running also as v1 mode in v2 sample test infra, so there's no need to test it here